### DISCUSSION TITLE
Is the analog counter necessary on this meter?

### DISCUSSION THREAD

**Discussion by odechr** [jomjol/AI-on-the-edge-device#2214] [Q&A]
Hi all,

I am about to install my first ESP32 cam on my water meter. 
And i have a question regarding the analog counter. Is this needed for me?
I have 3 digits on my meter. Looks like the analog counter is the same ast the last red digit?
![20221206_185058](https://user-images.githubusercontent.com/57597773/226468147-1bb93a95-70b5-47a2-ad9d-08c7b05bf093.jpg)

**Comment by caco3** [comment]
The analoge pointer is factor 10 smaller than your last digit.
Last digit represents m3: approx: 5.2
Analog counter value represents 0.1 m3: approx. 2.8

You do not need to use the analog counter, but it increases your accuracy
